standalone-metastore/metastore-server/pom.xml

```diff
@@ -395,6 +395,13 @@
       <version>2.32.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
```

Comment:
why is this change?

Reply:
Needed for the new unit tests added as part of this PR which use 
`import org.apache.logging.log4j.test.appender.ListAppender;`